app/queues/dns/workers/poll-dns-status-worker.server.ts

```diff
@@ -0,0 +1,39 @@
+import { Worker } from 'bullmq';
+import { redis } from '~/lib/redis.server';
+import logger from '~/lib/logger.server';
+import { getChangeStatus } from '~/lib/dns.server';
+import { dnsUpdateQueueName } from './dns-update-worker.server';
+import { getChildrenValuesOfQueueName } from '~/utils';
+
+export const checkDnsStatusQueueName = 'check-dns-status';
+
+// interface PollDnsStatus {
```

Comment:
Can we remove this?

Reply:
Sorry that I forgot removing the comments